resources/views/account/order.blade.php

```diff
@@ -1 +1,50 @@
-Order
+@extends('rapidez-ct::account.partials.layout')
+
+@section('title', __('Order') . ' #' . $id)
+@section('button')
```

Comment:
Since Blade components sections / layouts aren't used that much anymore, see the differences in the docs:
- 7.x: https://laravel.com/docs/7.x/blade#defining-a-layout
- 8.x: https://laravel.com/docs/8.x/blade#building-layouts

Maybe use that everywhere?

Reply:
Don't you think its a bit late for a refactor like this? Why hasn't this been seen earlier?

Reply:
I have created a story for this, to refactor the usage of layouts throughout the project :)